### DISCUSSION TITLE
medium and large creatures still aren't a threat

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#11301] [Ideas and suggestions]
enjoy this clip of a swarm of endworms failing to kill a couple AIs on an un-upgraded dugong for 11 [minutes:](https://youtu.be/qqEqbWx8voM)

the problem here is that the only characters that died are the ones dumb enough to go outside. a small amount of damage is given to characters close to the walls during an impact (through debris and impact) but it is never enough that 10 minutes of chip damage can't be healed with a single morphine

2 crawlers pose a larger threat than an endworm, which is a bit backwards
the only large creatures that pose a threat are the hammerhead matriarch (by way of creating lag) and the latcher (because it shoots spikes inside the sub)
an endworm can't even kill you by landing you on volcanos or below crush depth

i suggest that the wall shrapnel damage be dramatically increased, but dependent on force (maybe a hammerhead or moloch will deal 1/4 health in lacerations, but an endworm would deal 3/4)

perhaps a different type of shrapnel would be better, one that basically generates a metal "crystal" (like in caves) inside the sub that can be plasma cut, but injures you if you touch it

**Comment by ubertpendragon** [comment]
Allow me to elaborate on this

It's often said that Endworm/Charybdis are weaker than the Latcher, and that Endworm/Charybdis cannot kill the crew,
There are only 4 ways a monster can, in theory kill a crew inside a submarine:

1. Flooding by breaking walls and causing drowning/pressure
2. AoE damage by hitting the sub
3. Boarding the submarine and directly hitting the crew
4. Projectiles that hit the crew

Charybdis/Endworm only have 1 and 2

**Flooding is usually a non-hazard**: drowning is extremely hard in barotrauma; you can stay in your suit for a very long time, if you don't have a suit on you, you can leave one on the floor, the very existence of PUCS allows you to bypass oxygen consumption and stay suited forever. Since pressure is relative to your depth and suit, it is usually never a factor as well, even in the abyss.

**AoE damage is very low**: You can take blunt force trauma damage from these attack, but the affliction itself isnt caused by the attack, only by the force/stun. Hammerheads, Molochs, Endworm and Charybdis simply cannot kill a crew via blunt force. Only shrapnel from breaking walls can

The most effective forms of killing crew are Boarding and Projectiles, which none of Endworm/Charybdis have. 
What happens is, sessions can drag on for a very long time if a crew runs out of ammo and neither parties can kill the other.

Without either of the 4 causes those monsters end up being time wasters rather than hazards.

**Reply by inocent16** [reply]
but how do you get away for this situation? The sub is almost completly filled with water and you are not able to maintain your position. Istead you are sinking or falling into the abyss. Sooner or later you will run out of oxygen or fuel. So this is killing over time. The fact that the endworm makes so much hull damage is dangerous for these exakt reasons.

**Reply by girlyguppy** [reply]
> but how do you get away for this situation? The sub is almost completly filled with water and you are not able to maintain your position. Istead you are sinking or falling into the abyss. Sooner or later you will run out of oxygen or fuel. So this is killing over time. The fact that the endworm makes so much hull damage is dangerous for these exakt reasons.

Issue with that is that nobody has time nor wants to spend literally 2 hours welding leaks
So you basically just have to restart the mission or waste everyone's time

It's always better to just die

**Comment by BiffW** [comment]
tldr; Endworm and Charybdis needs some rework. Their attacks are not threatening enough.
I have an idea for Charybdis here. Add a soundwave attack for it that can deal AoE damage and penetrate the hull to damage the crew.

**Reply by BiffW** [reply]
Charybdis' weakness damage multiplier is 5. Although it has 8000 vitality, players can kill it by dealing 1600 damage to its mouth. 1600 vitality is only slightly higher than a moloch(1250 max vitality). I suggest an adjustment that lower the value to 2.5.

**Reply by girlyguppy** [reply]
Soundwave sounds cool
"Micro worms" is not

**Comment by BiffW** [comment]
For endworm. Maybe allow it to spawn some micro worms that can board the submarine to bite the crew.

**Reply by Koku0** [reply]
I don't think this would fit an Abyss monster. They're meant to be massive, threatening, solo encounters, not minion-spawners like the Hammerhead Matriarch.

**Comment by Rotiart-Drol** [comment]
I would add, for consideration, that spalling should be dramatically more lethal the more damaged a piece of hull is. You shouldn't be under threat from anything that can't enter the sub if the hull is in good repair, but if you allow the hull to fall into disrepair, you should be punished for it.

**Comment by OnestormGX1** [comment]
I'n my opinion Char and Worm have 2 big design flaws

-1.Timing and Knowledge: They appear very late into the campaign and are fully optional (wink), making 90% of their appareances just another fight for seasoned veterans with knowledge on how to supply, repair, heal and deal damage. 

-1.1.Other earlier monsters are a challenge for their ability to fight you with slow, paper subs with no supplies, or even a lack of crewmates and guns. and their special abilities (latcher), Char and worm are just a road raging Bus.

-2.Damage and tankyness: Being somewhat tanky gives them a sure edge (looking at you nerfed radiation poisoning) 
They may hit "hard", but they do it slowly, so you can repair their damage in less time they use to deal it, this is even worse for upgraded non-oneshoteable hulls

-2.1.If you can prolong the fight on the bottom of the ocean then it's only a fight of supplies, costing only ammo, welding tanks and medicines. And sometimes not even medicines (wink).

**Reply by Ek-Videogames** [reply]
Yeah exactly. In laymen's terms: 
Small enemies are fast, hard to see, and hard to hit, making it more likely that they will reach the hull.
_(and hard to shoot once they are on the hull as they can get wedged in crevices in the hull or under a turret's firing arc)_
They also are capable of entering the hull, an challenge which is further exacerbated by the very lackluster selection of small-arms the crew usually has access to during the early game that small enemies most often appear.

Large enemies are slow & easy to see, allowing the crew a non-zero amount of time to react to them, they are also a big easy to hit target. Sure they are often more tanky, but that difference tends to not mean much when a large monster is likely going to have the heaviest possible weaponry on the sub bearing down on it. Large enemies can't enter the sub, so at-worst it just becomes a battle of attrition and the crew can just hide in the submarine, and then spend 30 minutes repairing every wall & ever machine on the submarine once its all over.

**Comment by Hazmat616** [comment]
What if large creatures could damage turrets? This way they can disable you and force you that's to go outside to fix them.

**Comment by G-M-twostay** [comment]
maybe make the sea floor way lower. so that endworms pose threat by sinking the sub. currently sea floor is too shallow and when u think to the floor u r safe there.

**Comment by Hazmat616** [comment]
What if the moloch's could somehow flood the sub with acid. end worm could have an electrical discharge effect so it can electrocute the crew through breaches. Charibdis could create vacuums in the water to suck crew out through breaches as the creature in mythology could create whirlpools by sucking in water.
